lib/queue.js

```diff
@@ -1127,7 +1126,10 @@ Queue.prototype.clean = function(grace, type, limit) {
   }
 
   if (
-    _.indexOf(['completed', 'wait', 'active', 'delayed', 'failed'], type) === -1
+    _.indexOf(
+      ['completed', 'wait', 'active', 'paused', 'delayed', 'failed'],
```

Comment:
is there any tests on this? we should update the current tests or add new one if the test doesn't exist